src/Library/Environment/Android/AndroidEnvironment.cpp

```diff
@@ -0,0 +1,33 @@
+#include "AndroidEnvironment.h"
+
+#include <SDL.h>
+
+std::string AndroidEnvironment::queryRegistry(const std::string &path) const {
+    return {};
+}
+
+std::string AndroidEnvironment::path(EnvironmentPath path) const {
+    const char *result = nullptr;
+    if (path == PATH_ANDROID_STORAGE_INTERNAL) {
+        result = SDL_AndroidGetInternalStoragePath();
+    } else if (path == PATH_ANDROID_STORAGE_EXTERNAL) {
+        result = SDL_AndroidGetExternalStoragePath();
+    }
+
+    // TODO(captainurist): No PATH_HOME on Android? Verify & write a comment here.
```

Comment:
Each application on Android is executed under dedicated user so PATH_HOME is useless.
I think there are way to access application storage but I do not know the details.